packages/vuetify/src/components/VProgressLinear/VProgressLinear.ts

```diff
@@ -42,61 +54,101 @@ export default mixins(Colorable).extend({
     },
     height: {
       type: [Number, String],
-      default: 7
+      default: 4
     },
     indeterminate: Boolean,
     query: Boolean,
+    reactive: Boolean,
```

Comment:
I'm not sold on calling this prop `reactive`. It's too close to Vue's core concept of reactivity. All it does is make the progress bar clickable. So why not a name with `click` or `clickable` in it?